includes/User.php

```diff
@@ -3231,11 +3231,14 @@ public function saveSettings() {
 	 * @return Int
 	 */
 	public function idForName( $fromMaster = false ) {
+		global $wgExternalSharedDB;
 		$s = trim( $this->getName() );
 		if ( $s === '' ) return 0;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)

[project-id]: wikia:app
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)

[project-id]: wikia:app